ui/src/main/resources/edu/wpi/grip/ui/MainWindow.fxml

```diff
@@ -96,6 +96,23 @@
                     </accelerator>
                 </MenuItem>
             </Menu>
+            <Menu text="Tools">
+                <MenuItem text="Settings" onAction="#showProjectSettingsEditor">
+                    <graphic>
+                        <ImageView styleClass="menu-graphic">
+                            <fitWidth>
+                                <DPIUtility fx:constant="SMALL_ICON_SIZE"/>
+                            </fitWidth>
+                            <fitHeight>
+                                <DPIUtility fx:constant="SMALL_ICON_SIZE"/>
+                            </fitHeight>
+                            <image>
+                                <Image url="@icons/settings.png"/>
```

Comment:
Something like:
```xml
<graphic>
        <Glyph fontFamily="FontAwesome" icon="GEAR"/>
    </graphic>
```
https://bitbucket.org/controlsfx/controlsfx/pull-requests/381/basic-refactoring-of-glyph-font-system-306/diff